package.json

```diff
@@ -19,13 +19,19 @@
     "!ios/build",
     "!**/__tests__",
     "!**/__fixtures__",
-    "!**/__mocks__"
+    "!**/__mocks__",
+    "app.plugin.js",
+    "expo-module.config.json",
+    "plugin/build"
   ],
   "scripts": {
+    "build:plugin": "tsc --build plugin",
+    "clean:plugin": "expo-module clean plugin",
+    "lint:plugin": "eslint plugin/src/*",
```

Comment:
the existing `lint` command covers these files, right? Just wanna make sure it runs in CI